tokens/governance/treasury-spend.md

```diff
@@ -1,97 +1,103 @@
+    - **Title** and **Proposal Status** 
 
-To get started, head to [Polkadot.js Apps](https://polkadot.js.org/apps/?rpc=wss://wss.api.moonbase.moonbeam.network#/treasury){target=\_blank} and take the following steps:
+    - **Abstract** - two or three sentences that summarize the MBTP
 
-1. Select the **Governance** heading
-2. Click on **Treasury**
-3. Click **Submit Proposal**
+    - **Motivation** - a statement on why the Moonbeam Community via the Treasury Council should support the MBTP
 
-![Treasury home](/images/tokens/governance/treasury-proposals/treasury-proposal-2.webp)
+    - **Project Overview and Team Experience** - description of project and relevant team experience
 
-Then, take the following steps:
+    - **Rationale** - an explanation of how the MBTP would add value to the Moonbeam Ecosystem
+
+    - **Key Terms *(optional)*** - definitions of any terms within the proposal that are unique to the MBTP
+
+    - **Overall Cost** - the total cost of the MBTP and how this cost ties to the deliverables or milestones
+
+    - **Use of Treasury Funds** - detailed explanation of use of Treasury Funds including deliverables/ milestones and timeline
+
+    - **Specifications** - a detailed breakdown of the platforms and technologies that will be used
+
+    - **Steps to Implement** - the steps to implement the MBTP, including associated costs, manpower, and other resources for each step where applicable
+
+4. Press **Create Topic**
+
+![Submit a treasury spend proposal](/images/tokens/governance/treasury-proposals/treasury-proposal-2.webp)
+
+
+### Treasury Council Voting Process {: #treasury-council-voting-process }
+
+A member of the Treasury Council must first open [Polkadot.js Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fwss.api.moonbeam.network#/explorer){target=\_blank}  and navigate to the **Extrinsics** tab. From there, the treasury council member will submit a `treasury.spend` call. This call requires specifying the amount (for example, 1,000 GLMR) and the beneficiary account to receive the funds. Once this extrinsic is submitted, it creates a new Treasury Council collective proposal containing the specified spending information. Other members of the treasury council vote on the proposal with the `treasuryCouncilCollective.vote` extrinsic by providing a hash to the proposal, the proposal index, and aye/nay boolean to approve. Finally, a member of the treasury council closes the vote with the `treasuryCouncilCollective.close` extrinsic.
```

Comment:
This is just copied from the other page. We can just link to it instead of repeating it here.